notebook/debugging/data_similarity.ipynb

```diff
@@ -0,0 +1,1306 @@
+      "text/plain": [
+       "<Figure size 640x480 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "data": {
+      "text/plain": [
+       "<ggplot: (8791901875637)>"
+      ]
+     },
+     "execution_count": 37,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "plot = ggplot(means_df, aes(x='study_name', y='difference')) \n",
```

Comment:
I wonder if you could correct this for platform variance somehow - it seems like some platforms are going to tend to have lower or higher in-study mean pairwise distances just by virtue of their platform, not any study-specific variation (unless you're trying to get at differences between platforms, but it seems like maybe you want to detect outlying studies without the potential confounding effect of platform).

Here are a few ideas, up to you whether to pursue them or not:

* Could you regress out the platform (i.e. something super simple like what's described in [the intro to this paper](https://www.biorxiv.org/content/10.1101/2020.08.17.255034v1.abstract)), then do the same distance analysis? Or are there many different platforms such that each study is essentially a separate platform?
* Could you correct for changing variance between studies somehow? Maybe you could use something like the average [Mahalanobis distance](https://en.wikipedia.org/wiki/Mahalanobis_distance) comparing each point in the study with the mean of the study vs. the mean of the whole dataset would work for this, although I haven't thought this through in a super precise way.

Reply:
That's a good point, and may be something I was accidentally already doing. The initial results showed the effects after batch effect correction where I corrected by platform. I'm currently running the same analysis without be correction to see whether things change

Reply:
Got it! Yeah, I would imagine that BE correction would fix different variances per platform, but I'm not entirely sure. Comparing against the same analysis with no correction will definitely help clarify what's going on.

Reply:
The distributions look mostly the same when not batch effect correcting, except cross-study difference is much greater across the board